mlir/test/Dialect/GPU/subgroupId-rewrite.mlir

```diff
@@ -0,0 +1,26 @@
+// RUN: mlir-opt --test-gpu-rewrite -split-input-file %s | FileCheck %s
+
+module {
```

Comment:
This contains both an explicit module and and implicit one -- I don't think we need both. Can we drop either one?